src/passes/RemoveUnusedBrs.cpp

```diff
@@ -1360,6 +1381,14 @@ struct RemoveUnusedBrs : public WalkerPass<PostWalker<RemoveUnusedBrs>> {
             start++;
             continue;
           }
+          // If the first condition value is a tee, that is ok, so long as the
+          // others afterwards are gets of the value that is tee'd.
```

Comment:
Where is the check that the following tees are of the same value?

Reply:
We only allow the first item in a future `br_table` to be a tee - the other parts of the future `br_table` must be gets of the tee'd value.

That is enforced by only having tee logic up here (before the loop that finds the rest of the items), and by setting the condition that the others will be compared to to a get of the proper local.

(Separately from this PR, this pattern also has no "holes" in the middle, so nothing gets a chance to write to the local in between.)